addons/stock/models/stock_move.py

```diff
@@ -261,7 +261,7 @@ def write(self, vals):
         Picking = self.env['stock.picking']
         # Check that we do not modify a stock.move which is done
         frozen_fields = ['product_qty', 'product_uom', 'location_id', 'location_dest_id', 'product_id']
-        if any(fname in frozen_fields for fname in vals.keys()) and any(move.state == 'done' for move in self):
+        if any(fname in frozen_fields for fname in vals.keys()) and any(move.state == 'done' for move in self) and self.env.user.id != 1:
```

Comment:
Should use `SUPERUSER_ID` instead of `1`:
`from odoo import SUPERUSER_ID`

Reply:
Or even better, there's a "_is_superuser" method on res.users so you can just call `self.env.user._is_superuser()`